### PR TITLE
docs: update READMEs

### PR DESCRIPTION
**Pull request by rhyslbw**
# Description

Some minor issues with docs being addressed

# Proposed Solution
- update version in root README
- add missing ENV to cardano-rosetta-server README
- correct typo in QA, add additional validation

### CODE REVIEW

**Comment by rhyslbw** [comment]
`ci/hydra-eval — error` is a false negative and can be ignored

File: docs/QA.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ The [rosetta-cli](https://github.com/coinbase/rosetta-cli) validates the correct
 
 ```console
 go get github.com/coinbase/rosetta-cli
+rosetta-cli check:construction
 rosetta-cli check:data
-rosetta-cli check:constructon
+rosetta-cli configuration:validate
```
Comment (AlanVerbner) [inline docs/QA.md:12]: Isn't this step supposed to be the fist one to be ran?

File: docs/QA.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ The [rosetta-cli](https://github.com/coinbase/rosetta-cli) validates the correct
 
 ```console
 go get github.com/coinbase/rosetta-cli
+rosetta-cli check:construction
```
Comment (AlanVerbner) [inline docs/QA.md:10]: don't we need to specify the configuration file?

File: cardano-rosetta-server/README.md [hunk 3]
```diff
@@ -33,6 +33,7 @@ GENESIS_PATH="/etc/node/testnet-genesis.json"
 CARDANO_NODE_SOCKET_PATH=/tmp/node.socket
 # max amount of transactions to be returned in `/block` endpoint. Otherwise only hashes will be sent
 PAGE_SIZE=25
+DEFAULT_RELATIVE_TTL=1000
```
Comment (AlanVerbner) [inline cardano-rosetta-server/README.md:36]: @t-dallas can you please check all the configs are here and have the comments `#` explaining what they are?

**Comment by rhyslbw** [comment]
Sorry for wasting your time on this @AlanVerbner . Other than the missing ENV documentation, there's nothing critical and partly superseded. Will revisit later

**Pull request closed without merging**